Skype/UCWA/ApplicationDashboard.md

```diff
@@ -13,15 +17,16 @@ An [application](application_ref.md) resource represents a single application on
 
 The following table contains a representation of the **application** resource.
 
-
 **Property bag**
+
```

Comment:
did you add an empty line purposefully or was it created due to changing the format?

Reply:
Add an empty line to cleanly separate this content from the code-block.